libs/components/tiles/src/lib/modules/tiles/tile-dashboard/tile-dashboard.service.spec.ts

```diff
@@ -321,27 +322,28 @@ describe('Tile dashboard service', () => {
     }
   }
 
-  it('should allow tiles to be moved between columns with the keyboard', fakeAsync(() => {
+  it('should allow tiles to be moved between columns with the arrow keys', fakeAsync(() => {
     const fixture = createDashboardTestComponent();
     fixture.detectChanges();
     tick();
     fixture.detectChanges();
 
     // Check navigating to the right column
-    testIntercolumnNavigation(fixture, 'Right');
+    testIntercolumnNavigation(fixture, 'ArrowRight');
```

Comment:
> Edge (16 and earlier) and Firefox (36 and earlier) use "Left", "Right", "Up", and "Down" instead of "ArrowLeft", "ArrowRight", "ArrowUp", and "ArrowDown".

[MDN link](https://developer.mozilla.org/en-US/docs/Web/API/UI_Events/Keyboard_event_key_values#navigation_keys). Removing these to simplify tests.